src/main/java/aquality/appium/mobile/elements/actions/SwipeActions.java

```diff
@@ -0,0 +1,102 @@
+package aquality.appium.mobile.elements.actions;
+
+import aquality.appium.mobile.application.AqualityServices;
+import aquality.appium.mobile.configuration.ISwipeConfiguration;
+import aquality.appium.mobile.elements.ScreenSizeType;
+import aquality.appium.mobile.elements.SwipeDirection;
+import aquality.appium.mobile.elements.interfaces.IElement;
+import aquality.selenium.core.logging.Logger;
+import io.appium.java_client.TouchAction;
+import io.appium.java_client.touch.offset.PointOption;
+import org.openqa.selenium.Dimension;
+
+import static io.appium.java_client.touch.WaitOptions.waitOptions;
+
+public class SwipeActions {
+
+    /**
+     * returns new instance of TouchAction.
+     */
+    private static TouchAction<?> getTouchAction() {
```

Comment:
please, keep the order of private and public methods in the class. as a good style is to place public methods above privates. try do not mix them